### DISCUSSION TITLE
Add custom pages in the administration dashboard

### DISCUSSION THREAD

**Discussion by belhauss** [orchidsoftware/platform#1923] [Q&A] [answered]
I'm still new to the orchid platform, my question is can I add custom pages in the administrator dashboard to do some advanced actions with their views that cannot be done using orchid solution ? (charts and datatable.js with advanced and complicated editions)

**Comment by tabuna** [comment, marked as answer]
The easiest way is to call the template you need right on the screen:

```php
use Orchid\Support\Facades\Layout;

public function layout(): array
{
    return [
        Layout::view('myTemplate'),
    ];
}
```

You can find out more on this page: https://orchid.software/en/docs/custom-template/#views
